canonical/ubuntu.com
# Add missing classes in heading icon components

## Done

As part of preparation for typography updates I noticed quite a few inconsistencies in class names in heading icon component that were causing issues with upgrade.

This attempts to fix this by scoping down the CSS overrides added to `lazyloaded` div and adding all necessary class names for `p-heading-icon__img` and `p-heading-icon__title` elements that were missing across many instances of the component.

- Adds missing class names on elements in heading icon component
- Removes duplicated `__img` styles from `lazyloaded` div
- Removes unnecessary usage of heading icon component (where no icon was present)

## QA

Check all affected pages, make sure heading icon component is displayed as expected:

https://ubuntu-com-12807.demos.haus/0_docs/content_strips
https://ubuntu-com-12807.demos.haus/azure/sql
https://ubuntu-com-12807.demos.haus/blog/topics/robotics
https://ubuntu-com-12807.demos.haus/containers/shared/_whitepaper
https://ubuntu-com-12807.demos.haus/download/iot
https://ubuntu-com-12807.demos.haus/download/raspberry-pi
https://ubuntu-com-12807.demos.haus/download/raspberry-pi/thank-you
https://ubuntu-com-12807.demos.haus/edge-computing
https://ubuntu-com-12807.demos.haus/financial-services
https://ubuntu-com-12807.demos.haus/hpc
https://ubuntu-com-12807.demos.haus/internet-of-things/edgex
https://ubuntu-com-12807.demos.haus/internet-of-things/smart-city
https://ubuntu-com-12807.demos.haus/kubernetes/charmed-k8s
https://ubuntu-com-12807.demos.haus/kubernetes/compare
https://ubuntu-com-12807.demos.haus/kubernetes
https://ubuntu-com-12807.demos.haus/kubernetes/install
https://ubuntu-com-12807.demos.haus/kubernetes/managed
https://ubuntu-com-12807.demos.haus/kubernetes/what-is-kubernetes
https://ubuntu-com-12807.demos.haus/lxd
https://ubuntu-com-12807.demos.haus/managed/apps/kafka
https://ubuntu-com-12807.demos.haus/observability/managed
https://ubuntu-com-12807.demos.haus/openstack/consulting
https://ubuntu-com-12807.demos.haus/openstack/what-is-openstack
https://ubuntu-com-12807.demos.haus/security
https://ubuntu-com-12807.demos.haus/security/livepatch
https://ubuntu-com-12807.demos.haus/telco/osm/multi-cloud
https://ubuntu-com-12807.demos.haus/telco/osm/onboarding-network-functions

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
